contributors/guide/contributor-cheatsheet.md

```diff
@@ -13,7 +13,7 @@ A list of common resources when contributing to Kubernetes.
 - [Gubernator Dashboard - k8s.reviews](https://k8s-gubernator.appspot.com/pr)
 - [reviewable.kubernetes.io](https://reviewable.kubernetes.io/reviews#-)
 - [Submit Queue](https://submit-queue.k8s.io)
-- [Bot commands](https://git.k8s.io/test-infra/commands.md)
+- [Bot commands](https://prow.k8s.io/command-help)
```

Comment:
Can you please change this to link to `https://go.k8s.io/bot-commands`? This redirect can be pointed somewhere else if the URL changes again.

Reply:
okay, that's even better!